drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -12,6 +12,8 @@ menuconfig ETH_SAM_GMAC
 	select NOCACHE_MEMORY if ARCH_HAS_NOCACHE_MEMORY_SUPPORT
 	select ETH_DSA_SUPPORT_DEPRECATED
 	select PINCTRL
+	imply NVMEM if ($(dt_compat_any_has_prop,$(DT_COMPAT_ATMEL_SAM_GMAC),nvmem-cells) || \
```

Comment:
`select` or `imply`?

Reply:
I prefer `imply` as it isn't a hard requirement, and could be turned off, for example for testing purposes.